openpgp.d.ts

```diff
@@ -577,7 +577,7 @@ interface EncryptOptions {
   /** (optional) session key */
   sessionKey?: SessionKey;
   /** if the return values should be ascii armored or the message/signature objects */
-  armor?: boolean;
+  format?: 'armor' | 'binary';
```

Comment:
After staring at this PR for a while, I do think that `format: 'armored'` indeed makes a bit more sense :sweat_smile:
The comment here kind of confirms the point :upside_down_face: 
Since it's the default, the fact that it's a bit longer shouldn't really matter.

Reply:
I will open a new PR to rename this also in the openpgp.*Key functions 👍